lib/Catalyst/Plugin/Sentry.pm

```diff
@@ -0,0 +1,167 @@
+package Catalyst::Plugin::Sentry;
+
+use v5.10;
+use strict;
+use warnings;
+
+use DBDefs;
+use Devel::StackTrace;
+use File::Slurp qw( read_file );
+use IO::File;
+use Moose;
+use Sentry::Raven;
+
+our $suppress = 0;
+
+sub execute {
```

Comment:
A lot of what this does should in theory be [handled by Sentry::Raven already](https://github.com/getsentry/perl-raven/blob/291fc8444258c2ea6112206051550ab036812fa7/lib/Sentry/Raven.pm#L229). Unfortunately, I haven't yet found a way to get that to work with Catalyst, but I'll keep trying.